### PR TITLE
Add recipe for installing version 0.11.4 of the FastQC package

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: packages/package_fastqc_0_11_4/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                        <destination_directory>$INSTALL_DIR</destination_directory>
+                        <action type="shell_command">chmod ugo+x $INSTALL_DIR/fastqc</action>
+                    </action>
+                </actions>
+                <actions os="darwin" architecture="x86_64">
+                    <action type="download_by_url" sha256sum="bd4089397cfa1086d81e7b29508542d2478917df8d1ef1fb494616da1b8e651a">https://depot.galaxyproject.org/software/fastqc/fastqc_0.11.4_darwin_all.dmg</action>
+                    <action type="shell_command">
+                        hdiutil attach fastqc_v0.11.4.dmg &amp;&amp;
+                        cp -R /Volumes/FastQC/FastQC.app $INSTALL_DIR &amp;&amp;
+                        hdiutil detach /Volumes/FastQC
```
Comment (bgruening) [inline packages/package_fastqc_0_11_4/tool_dependencies.xml]: I have no clue what this is, I just hope this is portable enough! Thanks Greg!

**Comment by bgruening** [comment]
@gregvonkuster can you please add a `.shed.yml` as well?

**Comment by gregvonkuster** [comment]
Sure, just a sec...

File: packages/package_fastqc_0_11_4/tool_dependencies.xml (hunk 1 above)
Comment (gregvonkuster) [inline packages/package_fastqc_0_11_4/tool_dependencies.xml]: @bgruening the hdutil call handles .dmg files on OS X.  I promise it works!  ;)

**Pull request merged**